.github/workflows/release.yaml

```diff
@@ -0,0 +1,338 @@
+name: release
+
+# TODO - the real trigger
+# on:
+#   push:
+#     tags:
+#       - 'v*'
+
+# FAKE for testing
+on:
+  pull_request:
+    paths:
+      - '**/*'
+      - '!docs/**'
+      - '!examples/**'
+      - '!README.md'
+
+
+# TODO - wire up signing for official releasese
+
+jobs:
+  # setup the version for easy consumption in the other jobs
+  version:
+    runs-on: ubuntu-latest
+    steps:
+    # - uses: 'google-github-actions/auth@v2'
+    #   with:
+    #     project_id: 'ollama'
+    #     workload_identity_provider: '${{ secret goes here }}'
+    # - run: |
+    #     gcloud projects list
+    - name: Get the target release version
+      id: get_version
+      run: |
+        ver=${GITHUB_REF/refs\/tags\//}
+        echo "VERSION=0.0.0" >> "$GITHUB_OUTPUT"
+      #  echo "VERSION::${ver#v}" >> "$GITHUB_OUTPUT"

```

Comment:
```suggestion
```

Reply:
While I'm testing this CI run, it's triggering off a PR, so the line 35 logic doesn't yield a version tag, hence I'm hard-coding it to 0.0.0.  Once this is wired up to trigger off a legit tag, then line 37 is the correct setup to get the version wired up.